### PR TITLE
Issue-176-ReactionRoles

### LINKED ISSUE #176: Reaction Roles
Maybe using something like [this](https://github.com/NKN1396/Discord-EmojiToRole)?

### PR DESCRIPTION
**Pull request by anubhav06**
# Pull Request

## Description

Added Reaction Role Command. 
User can create an embed and then add the reaction-role pair as required by the user. Command usage is mentioned
in the file `reaction.js`

## Related Issue

Closes #176 

## Scope

Did you remember to update the `package.json` version number?

- [ ] Major Version Update: X.\_.\_ (for complete code refactors, or large PRs that adjust most of the codebase)
- [x] Minor Version Update: \_.X.\_ (for the addition of new commands)
- [ ] Patch Version Update: \_.\_.X (for bug fixes _in the code_.)
- [ ] No Version Update: \_.\_.\_ (no version update for additions to tests, documentation, or anything that isn't end-user facing.)

## Documentation

For _any_ version updates, please verify if the documentation page needs an update. If it does, please create an issue to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "heptagram",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (j-dogcoder) [inline package.json:3]: Can this move to 1.2.0? We have another PR before this that needs to be merged.

**Review by j-dogcoder — CHANGES_REQUESTED** [review]

**Comment by j-dogcoder** [comment]
@Mergifyio rebase

**Comment by j-dogcoder** [comment]
@Mergifyio rebase

**Comment by anubhav06** [comment]
Do I need to make any changes to the commits or is it resolved ?

**Comment by j-dogcoder** [comment]
It is resolved, I just need to merge it. I will do that super soon.

**Review by j-dogcoder — APPROVED** [review]

**Comment by j-dogcoder** [comment]
LGTM @anubhav06! 🎉

File: commands/Resources and Utilitys/reaction.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+			// Check for each 'reaction=role' pair
+			for(const reaction of reactions) {
+				if (reaction.includes('=')) {
+					const split = reaction.split('=');
+					const emoji = split[0].trim();
+					const role = split[1].trim();
+
+					if (!message.guild.roles.cache.find(obj => obj.name === role)) {
+						message.reply(`The mentioned role: "${role}" does not exist`);
```
Comment (Panquesito7) [inline commands/Resources and Utilitys/reaction.js:61]:
```suggestion
						message.reply(`The role "${role}" does not exist`);
```

File: commands/Resources and Utilitys/reaction.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+			});
+
+		}
+		catch(error) {
+			console.log(error);
+		}
+
+
+	},
+};
```
Comment (Panquesito7) [inline commands/Resources and Utilitys/reaction.js:115]: Missing an endline.
```suggestion
};

```

**Comment by Panquesito7** [comment]
Was this PR reverted? I do not see this anywhere in the repository. 👀

**Comment by j-dogcoder** [comment]
It is staged in the stage branch, just not on master yet @Panquesito7

**Pull request merged**